src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleItem/ScheduleItem.tsx

```diff
@@ -39,12 +43,19 @@ const ScheduleItem: React.FC<IScheduleItemProps> = ({
   useMemo(() => stateSchedule.zeroing && setChoiceDate(choiceDate), [stateSchedule.zeroing]);
 
   return (
-    <ScheduleItemWrapper disabled={disabled} onClick={() => itemSelected()} choice={choiceDate}>
+    <ScheduleItemWrapper
+      disabled={disabled}
+      onClick={() => {
+        itemSelected();
+        setDisabledButton(!disabledButton);
```

Comment:
can we extract this logic to a separate function?

Reply:
fixed